src/cpp/src/llm_pipeline.cpp

```diff
@@ -266,11 +266,16 @@ class StatefulLLMPipeline final : public LLMPipelineImplBase {
             m_history = {};
             m_templated_chat_history = "";
         }
+        OPENVINO_ASSERT(m_tokenizer.is_chat_template_ready(),
+                        "There is no existing chat template for actual model. LLMPipeline cannot work in chat mode."
+                        " Please add chat template to tokenizer_config.json or use another model.");
```

Comment:
Please, suggest "default" content for "chat_template" entry in the error message with a disclaimer that it may not be suitable for the model.